### DISCUSSION TITLE
PHP Warning:  stream_socket_enable_crypto() [<a href='function.stream-socket-enable-crypto'>function.stream-socket-enable-crypto</a>]: SSL: Success in /home/hideysco/public_html/php/phpmailer/class.smtp.php on line 254

### DISCUSSION THREAD

**Discussion by maxhideys** [PHPMailer/PHPMailer#2543] [Q&A]
Hello I am getting this error and was wondering if anyone knows how I might be able to fix it. My website has stopped sending emails. 

`
PHP Warning:  stream_socket_enable_crypto() [<a href='function.stream-socket-enable-crypto'>function.stream-socket-enable-crypto</a>]: SSL: Success in /phpmailer/class.smtp.php on line 254`

This is line 254:
`
    if(!stream_socket_enable_crypto($this->smtp_conn, true, STREAM_CRYPTO_METHOD_TLS_CLIENT)) {
`
I am currently running 5.2.6

**Comment by Synchro** [comment]
Here's the problem:

> I am currently running 5.2.6

That version is many years out of date. Update and try again.
